docs/DeveloperGuide.md

```diff
@@ -294,18 +294,18 @@ The command takes in:
 - ```Set<Interest>```
 
 #### Implementation
-Each ```Person``` has a ```Set``` of ```Interest``` and adding an ```Interest``` would add the specified ```Interest``` into this set.
+Each ```Student``` has a ```Set``` of ```Interest``` and adding an ```Interest``` would add the specified ```Interest``` into this set.
 
 The Add Interest mechanism is facilitated by ```AddInterestCommand```, which extends from ```Command``` and ```AddInterestCommandParser```, which extends from
 ```Parser```. ```AddInterestCommandParser``` serves to parse the command arguments and create a new ```AddInterestCommand``` object. ```AddInterestCommand```handles adding the ```Interest``` to the set of ```Interest``` of the ```Person```.
```

Comment:
... to the set of `Interest` of the `Student`?
Actually, a `Student` is the user who adds each `Batchmate` to Mass Linkers. 🤔 💭